Description: don't link libpython
 This patch removes dependency to libpython 
Author: Matthias Klose <doko@ubuntu.com>
Origin: http://launchpadlibrarian.net/166876566/py3cairo_1.10.0%2Bdfsg-3build2_1.10.0%2Bdfsg-3ubuntu1.diff.gz
Bug-Debian: http://bugs.debian.org/739607
Last-Update: 2014-02-20

--- py3cairo-1.10.0+dfsg.orig/waflib/Tools/python.py
+++ py3cairo-1.10.0+dfsg/waflib/Tools/python.py
@@ -155,8 +155,10 @@ def check_python_headers(conf):
 		if result:
 			break
 	if result:
-		env['LIBPATH_PYEMBED']=path
-		env.append_value('LIB_PYEMBED',[name])
+		# don't link libpython explicitly
+		#env['LIBPATH_PYEMBED']=path
+		#env.append_value('LIB_PYEMBED',[name])
+		pass
 	else:
 		conf.to_log("\n\n### LIB NOT FOUND\n")
 	if(sys.platform=='win32'or sys.platform.startswith('os2')or sys.platform=='darwin'or dct['Py_ENABLE_SHARED']):
@@ -278,4 +280,4 @@ feature('pyembed')(init_pyembed)
 conf(get_python_variables)
 conf(check_python_headers)
 conf(check_python_version)
-conf(check_python_module)
\ No newline at end of file
+conf(check_python_module)
